administrator/components/com_admin/sql/updates/mysql/5.3.0-2025-01-23.sql

```diff
@@ -0,0 +1,4 @@
+UPDATE `#__content_types`
+SET `field_mappings` = JSON_SET(`field_mappings`, '$.common.core_created_user_id', 'created_by'),
+    `field_mappings` = JSON_SET(`field_mappings`, '$.common.core_created_by_alias', 'created_by_alias')
+WHERE `type_alias` = 'com_content.article';
```

Comment:
```suggestion
UPDATE `#__content_types`
SET `field_mappings` = JSON_SET(`field_mappings`, '$.common.core_created_user_id', 'created_by'),
WHERE `type_alias` = 'com_content.article';

UPDATE `#__content_types`
SET `field_mappings` = JSON_SET(`field_mappings`, '$.common.core_created_by_alias', 'created_by_alias')
WHERE `type_alias` = 'com_content.article';
```
We should use separate update stamenets for each modification to avoid issues and to be in synch with PostgreSQL where we have to do that to avoid an SQL error.